types/index.d.ts

```diff
@@ -142,7 +142,7 @@ type MatchType = "begin" | "end" | "illegal"
 
  interface ModeCallbacks {
      "on:end"?: Function,
-     "on:begin"?: Function,
+     "on:begin"?: (match: RegExpMatchArray, response: Response) => void
```

Comment:
@joshgoebel Why didn't you use `type ModeCallback` from line 125 here? The problem with `Response` is that for me this resolves to `lib.dom.d.ts` which also has a `Response`.

Reply:
Forgot about it... Is that better?

Reply:
Not only better, it resolves the linter issue on my side entirely. Thanks.